coalib/results/Diff.py

```diff
@@ -221,7 +221,7 @@ def affected_code(self, filename):
         return list(diff.range(filename)
                     for diff in self.split_diff(distance=0))
 
-    def split_diff(self, distance=1):
+    def split_diff(self, distance=-1):
```

Comment:
@adtac `Distance` is the `number of lines` allowed between two diffs.
```
z=0
a=1
b = 2
c=1
```
If you run PEP8Bear on the above code, it will generate 3 diffs which are separated by a single line (Line 1,2,4). If you run `split_diff(distance=-1)` on the original `Diff` object you'll get something like `[ [1], [2], [3] ]` i.e 3 values. If you run `split_diff(distance=0)` on the original `Diff` you'll get `[ [1,2], [3] ]` i.e 2 values and if you run `split_diff(distance=1)` you'll get `[ [1,2,3] ]` i.e 1 value.